assets/inference/environments/minimal-app-quickstart/context/conda_dependencies.yaml

```diff
@@ -8,7 +8,7 @@ dependencies:
 - pip:
   # vulnerabilities
   - cryptography>=41.0.3
```

Comment:
remove cryptography to wheel, there should be no need having those in conda spec

Reply:
removed